src/index.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+// const axios = require('axios');
+
+// changing color of temperature value
+const tempColor = (temp) => {
+  if (temp >= 80) {
+    document.getElementById('temp-value').style.color = 'red';
```

Comment:
Ideally this would set a class and be styled with CSS (using a class like "hot") instead of setting the color directly.

This would give us much more flexiblity with styling.